html/elements/details.json

```diff
@@ -0,0 +1,111 @@
+{
+  "html": {
+    "elements": {
+      "details": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/details",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "12"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": false,
+              "notes": "<a href='https://developer.microsoft.com/en-us/microsoft-edge/platform/status/detailssummary?filter=f3f0000bf&search=details'>Under consideration</a>."
+            },
+            "edge_mobile": {
+              "version_added": false,
+              "notes": "<a href='https://developer.microsoft.com/en-us/microsoft-edge/platform/status/detailssummary?filter=f3f0000bf&search=details'>Under consideration</a>."
+            },
+            "firefox": {
+              "version_added": "49"
```

Comment:
There has been a note added here in the meantime about Firefox 57 / Quantum. Do we want to add it? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#Quantum_CSS_notes

Reply:
Fixed. I rephrased it to merely state Firefox 57 (rather than Quantum that is an internal-name only).

Also, Stylo will not be in Firefox 57 for Android (I think it will be in Fx 58 or 59), so I rephrased the comment for Android to state that there still is the stated bug.